pkg/fatfs/fatfs_vfs/fatfs_vfs.c

```diff
@@ -29,6 +29,9 @@
 #include "fs/fatfs.h"
 
 #include "time.h"
+#if CONFIG_FATFS_FORMAT_ALLOC_STATIC
```

Comment:
No harm in always including the header

Reply:
Addressed. I force-pushed with a new commit message.